README.md

```diff
@@ -87,6 +87,23 @@ Viewing word clouds doesn't require login access. The URL of the view page is in
         font size, the higher the voice pitch.
         - **"Apply conventional weighting"**: When enabled, more frequently submitted words will be highlighted.
 
+### Simulate a Word Cloud
```

Comment:
Once the answers file is created, shall one more step be added to explain what to do with this file? Such as issuing a pull request to commit the generated file into the repository, then the provided answers will be shown on the Wordles site for that specific question.